models/config.py

```diff
@@ -16,5 +16,7 @@ def get_all(self):
 	def get_pubnub_keys(self):
 		return self.base_request.request('/config', 'GET', endpoint=self.settings.get('pine_endpoint'))['pubnub']
 
-	def get_device_type(self):
-		return self.base_request.request('/config', 'GET', endpoint=self.settings.get('pine_endpoint'))['deviceTypes']
+	def get_device_types(self):
```

Comment:
You could do it even better with a lazy property:

    @property
    def config(self):
      if not hasattr(self, '_config'):
        self._config = self.base_request.blah()
      return self._config

And you can use it like so:

    foo = self.config['bar']